### PR TITLE
bumping version from 1.5.0 -> 1.6.0

### PR DESCRIPTION
**Pull request by ryandeivert**
to: @jacknagz or @chunyong-lin 
cc: @airbnb/streamalert-maintainers
size: small
resolves N/A

## Changes

* Version bump from `1.5.0` --> `1.6.0`

### CODE REVIEW

File: stream_alert_cli/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 """Define the version of the CLI."""
-__version__ = '1.4.0'
```
Comment (jacknagz) [inline stream_alert_cli/__init__.py]: should this just point to the main lib version?

Comment (ryandeivert) [reply stream_alert_cli/__init__.py]: I wasn't sure if there was a reason the cli was versioned separately. should we remove this?

Comment (jacknagz) [reply stream_alert_cli/__init__.py]: yeah let's just keep them consistent for now.

**Comment by austinbyers** [comment]
If you want to add a version flag @ryandeivert , it's very easy: https://github.com/airbnb/binaryalert/pull/84

**Comment by ryandeivert** [comment]
@austinbyers done!! thanks :)

**Pull request merged**